### ISSUE TITLE
[Bug]: inappmessage modal - icon & text not centered

### ISSUE THREAD

**Issue by kosiara** [braze-inc/braze-flutter-sdk#92]
### Which Platforms?

Android

### Which Flutter Version?

3.24.0

### Which Braze Flutter SDK version?

11.1.0

### Repro Rate

100% of time

### Steps To Reproduce

Launch inapp message on Android

If an icon is placed it is moved to the left - instead of being centered. 
Title text & content text  is moved to the left - instead of being centered. 

![Screenshot 2024-10-23 at 11 27 01 AM](https://github.com/user-attachments/assets/fd02ee31-7544-4344-8066-6779e2b04d0a)

### Expected Behavior

In inapp message on Android the icon should be centered. Title text & content text should be centered.

![Screenshot 2024-10-23 at 11 29 23 AM](https://github.com/user-attachments/assets/ddc1c120-2cca-4c35-a63c-2d1b5dbd6d2e)

### Actual Incorrect Behavior

There's a bug in 
BRAZE_SDK_VERSION=32.1.0
the ISSUE is fixed in BRAZE_SDK_VERSION=33.0.0

### Additional Information

Modal_icon should NOT have               `android:textAlignment="viewStart"` attribute. 

```
<RelativeLayout
            android:id="@+id/com_braze_inappmessage_modal_image_layout"
            style="@style/Braze.InAppMessage.Layout.Image.Modal">
            <com.braze.ui.inappmessage.views.InAppMessageImageView
                android:id="@+id/com_braze_inappmessage_modal_imageview"
                style="@style/Braze.InAppMessage.Image.Modal"/>
            <TextView
                android:id="@+id/com_braze_inappmessage_modal_icon"
                android:textAlignment="viewStart"
                style="@style/Braze.InAppMessage.Icon.Modal"/>
        </RelativeLayout>
```

The issue was fixed in `BRAZE_SDK_VERSION=33.0.0` here:
https://github.com/braze-inc/braze-android-sdk/commit/f0df61c8afd1c283a81afd9272b9662cd6ec1694#diff-b507fa487f229614c138a1a396fc32016ea1b551700b83fdc112758b059c7e7dR33

Also bumping the SDK version to `BRAZE_SDK_VERSION=33.0.0`
causes a conflict in Kotlin version for a default flutter project. 

https://github.com/braze-inc/braze-flutter-sdk/compare/master...kosiara:braze-flutter-sdk:master

```
/Users/.pub-cache/git/braze-flutter-sdk-f9fa4130745197ac5fa79ee5c80b9a21bae89bfd/android/src/main/kotlin/com/braze/brazeplugin/
BrazePlugin.kt: (549, 48): 
Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. 
The binary version of its metadata is 1.9.0, expected version is 1.7.1.

e: /Users/.pub-cache/git/braze-flutter-sdk-f9fa4130745197ac5fa79ee5c80b9a21bae89bfd/android/src/main/kotlin/com/braze/brazeplugin
/BrazePlugin.kt: (549, 21): Class 'com.braze.Braze.Companion' 
was compiled with an incompatible version of Kotlin. 
The binary version of its metadata is 2.0.0, expected version is 1.7.1.
```

**Comment by hokstuff** [comment]
Hi @xleon,

Thanks for bumping this thread! We have prepared a release branch in our private repository and are validating it to release a new Flutter SDK version shortly.

**Comment by vanessaland** [comment]
Hi @kosiara and @xleon,

We have just released our [Braze Flutter SDK version 12.0.0](https://github.com/braze-inc/braze-flutter-sdk/releases/tag/12.0.0), which includes the latest version of our Android SDK.

For more information, see [the changelog](https://pub.dev/packages/braze_plugin/changelog).

Thank you!
